tmk_core/protocol/vusb/main.c

```diff
@@ -6,35 +6,38 @@
  * Copyright: (c) 2008 by OBJECTIVE DEVELOPMENT Software GmbH
  * License: GNU GPL v2 (see License.txt), GNU GPL v3 or proprietary (CommercialLicense.txt)
  * This Revision: $Id: main.c 790 2010-05-30 21:00:26Z cs $
+ *
+ * Modify for QMK firmware
+ * Copyright 2020 Yiancar
+ * Copyright 2020 Takuya Urakawa (dm9records.com)
  */
 #include <stdint.h>
 #include <avr/interrupt.h>
 #include <avr/wdt.h>
 #include <avr/sleep.h>
 #include <util/delay.h>
 #include "usbdrv.h"
-#include "oddebug.h"
 #include "vusb.h"
 #include "keyboard.h"
 #include "host.h"
 #include "timer.h"
 #include "uart.h"
 #include "debug.h"
 
-#if defined(RGBLIGHT_ENABLE)
+#if (defined(RGB_MIDI) || defined(RGBLIGHT_ANIMATIONS)) && defined(RGBLIGHT_ENABLE)
 #    include "rgblight.h"
 #endif
```

Comment:
```suggestion
```
In #7733, the call to `rgblight_task()` was removed from `protocol/vusb/main.c`, so this import should no longer be necessary.